### PR TITLE
Update gradients

### PR DESCRIPTION
**Pull request by jo-tomlinson**

### CODE REVIEW

**Comment by rosswarren** [comment]
Changing the gradients to linear, and changing the colours should be two separate PRs I think.

This is because the colours needs to be changed on the 19th of April but we can change them to linear now.

https://github.com/sky-uk/sky-pages/issues/793
vs
https://github.com/sky-uk/sky-pages/issues/795

File: src/styles/_mixins.scss [hunk 1]
```diff
@@ -1,7 +1,7 @@
 @mixin page-background-gradient($info) {
     background-color: nth($info, length($info));
-    background-image: -webkit-radial-gradient(nth($info, 1), nth($info, 2));
-    background-image: radial-gradient(nth($info, 1), nth($info, 2));
+    background-image: linear-gradient(to top, nth($info, 1), nth($info, 2));
```
Comment (beclamide) [inline src/styles/_mixins.scss:3]: Should reference the 3rd colour in the array, and gradient from direction specified in direction property
`background-image: linear-gradient(to $direction, nth($info, 1), nth($info, 2), nth($info, 3));`

File: src/styles/_mixins.scss [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+    background-image: -webkit-linear-gradient(90deg, nth($info, 1), nth($info, 2));
```
Comment (beclamide) [inline src/styles/_mixins.scss:4]: Should reference the 3rd colour in the array
`background-image: -webkit-linear-gradient(0, nth($info, 1), nth($info, 2), nth($info, 3));`

File: src/styles/_constants.scss [hunk 3]
```diff
@@ -1,11 +1,11 @@
 $polaris-bg-default: #FFFFFF, #E1E0E4, #FFFFFF;
```
Comment (beclamide) [inline src/styles/_constants.scss:1]: Should be grey to white to grey. (ref: https://app.zeplin.io/project.html#pid=56cf0a7e3173092129bdfcf4&sid=56cf0adaddc4c65742e414de)
`$polaris-bg-default: #e5e5ea, #FFFFFF, #e5e5ea;`

**Comment by beclamide** [comment]
Also the Base64 background image in gradients.scss needs updating to match the new gradient. The new code is below:

```
background:url(data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiA/Pg0KPHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxMDAlIiBoZWlnaHQ9IjEwMCUiIHZpZXdCb3g9IjAgMCAxIDEiIHByZXNlcnZlQXNwZWN0UmF0aW89Im5vbmUiPg0KICA8bGluZWFyR3JhZGllbnQgaWQ9ImdyYWQtdWNnZy1nZW5lcmF0ZWQiIGdyYWRpZW50VW5pdHM9InVzZXJTcGFjZU9uVXNlIiBjeD0iNTAlIiBjeT0iNTAlIiByPSI3NSUiPg0KICAgIDxzdG9wIG9mZnNldD0iMCUiIHN0b3AtY29sb3I9IiNlNWU1ZWEiIHN0b3Atb3BhY2l0eT0iMSIvPg0KICAgIDxzdG9wIG9mZnNldD0iNTAlIiBzdG9wLWNvbG9yPSIjZmZmZmZmIiBzdG9wLW9wYWNpdHk9IjEiLz4NCiAgICA8c3RvcCBvZmZzZXQ9IjEwMCUiIHN0b3AtY29sb3I9IiNlNWU1ZWEiIHN0b3Atb3BhY2l0eT0iMSIvPg0KICA8L2xpbmVhckdyYWRpZW50Pg0KICA8cmVjdCB4PSItNTAiIHk9Ii01MCIgd2lkdGg9IjEwMSIgaGVpZ2h0PSIxMDEiIGZpbGw9InVybCgjZ3JhZC11Y2dnLWdlbmVyYXRlZCkiIC8+DQo8L3N2Zz4=);
```

File: src/styles/_mixins.scss [hunk 4]
```diff
@@ -1,7 +1,7 @@
 @mixin page-background-gradient($info) {
```
Comment (Beclamide) [inline src/styles/_mixins.scss]: Duplicate and rename to `page-background-gradient-horizontal`

Comment (beclamide) [inline src/styles/_mixins.scss:1]: Add second property for direction?

**Comment by dtt101** [comment]
Hi - so you need to close this pull request and rework. See https://github.com/sky-uk/sky-pages/issues/788

You need one pull request for the gradient and one for the colour change.

**Comment by jo-tomlinson** [comment]
Splitting pull request to 2 separate requests for gradient and colour changes.

**Pull request closed without merging**